lib/src/DateTimePicker/DateTimePickerTabs.tsx

```diff
@@ -40,7 +40,8 @@ export const useStyles = makeStyles(
         : theme.palette.background.default;
 
     return {
-      container: {
+      root: {},
+      tabsDesktop: {
```

Comment:
I don't think that we should include tabs in the key, it's redundant: `.MuiDateTimePickerTabs-tabsDesktop`. Maybe?

```suggestion
      desktop: {
```

Or to match the `desktopModeMediaQuery` prop? 

```suggestion
      modeDesktop: {
```